OpenRA.Mods.D2k/Activities/DeliverUnit.cs

```diff
@@ -24,9 +24,10 @@ public class DeliverUnit : Activity
 		readonly Carryable carryable;
 		readonly Carryall carryall;
 		readonly Helicopter helicopter;
-		readonly IPositionable positionable;
+		readonly Mobile mobile;
```

Comment:
Does this really need to be `Mobile`? That restricts the type of unit that can be delivered.